tests/log/test_log.py

```diff
@@ -0,0 +1,6 @@
+from decimal import Decimal
+
+
+def test_log(w3, LOG):
```

Comment:
To thouroughly test the algorithm I suggest using hypthosis here, see https://github.com/ethereum/vyper/blob/master/tests/parser/types/numbers/test_sqrt.py#L145